samtranslator/feature_toggle/feature_toggle.py

```diff
@@ -13,9 +13,6 @@
 )
 from samtranslator.metrics.method_decorator import cw_timer
 
-my_path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Why we needed this before?

Reply:
@mndeveci See discussion here: https://github.com/aws/serverless-application-model/pull/2076#discussion_r796798622

TLDR: Was due to the test script. Since this is clobbering things on customers machines, I removed feature_toggle from the test script. If we really need it there (which I don't think we do), we can find a different way to add it back.